src/features/rewards/printableActivity/style.ts

```diff
@@ -0,0 +1,171 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License. v. 2.0. If a copy of the MPL was not distributed with this file.
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import styled from '../../../theme'
+
+export const StyledWrapper = styled<{}, 'div'>('div')`
+  background-color: white;
+`
+
+export const StyledMarginWrapper = styled<{}, 'div'>('div')`
+  font-family: ${p => p.theme.fontFamily.body};
+  margin: 50px;
+`
+
+export const StyledTitle = styled<{}, 'div'>('div')`
+  font-weight: 600;
+  color: #1B1D2F;
+  font-family: ${p => p.theme.fontFamily.heading};
+  font-size: 16px;
+  line-height: 2;
+`
+
+export const StyledSubTitle = styled<{}, 'span'>('span')`
+  color: #838391;
+  font-weight: normal;
+`
+
+export const StyledHeader = styled<{}, 'div'>('div')`
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+`
+
+export const StyledLeft = styled<{}, 'div'>('div')`
+  flex-basis: 40%;
+`
+
+export const StyledRight = styled<{}, 'div'>('div')`
+  flex-basis: 378px;
+  flex-grow: 0;
+  flex-shrink: 1;
+  margin-bottom: 45px;
+`
+
+export const StyledSelectOption = styled<{}, 'div'>('div')`
+  font-size: 22px;
+  font-weight: 300;
+  color: #4C54D2;
+`
+
+export const StyledIconWrap = styled<{}, 'div'>('div')`
+  margin-bottom: 103px;
+  display: flex;
+`
+
+export const StyledIcon = styled<{}, 'button'>('button')`
+  display: flex;
+  margin-right: 35px;
+  background: none;
+  border: none;
+  cursor: pointer;
+  align-items: center;
+`
+
+export const StyledIconText = styled<{}, 'div'>('div')`
+  font-size: 14px;
+  line-height: 1.43;
+  color: #838391;
+  margin-left: 13px;
+`
+
+export const StyledBalance = styled<{}, 'div'>('div')`
+  margin-top: 41px;
+`
+
+export const StyledTables = styled<{}, 'div'>('div')`
+  margin: 0 -50px;
+  padding: 0 50px;
+`
+
+export const StyledWarning = styled<{}, 'div'>('div')`
+  display: flex;
+  justify-content: center;
+  border-top: 1px solid #ebecf0;
```

Comment:
please use theme color

Reply:
++ If there is not a generic and shareabel theme color, then we can create one after asking design team @jenn-rhim @rossmoody, e.g. `warningBackgroundBorder`.

If this is a rewards-specific color rule then maybe we should start a rewards theme.